### PR TITLE
Updating FT.SEARCH documentation with search guarantee language

### PR DESCRIPTION
**Pull request by chayim**

### CODE REVIEW

**Comment by chayim** [comment]
@nermiller Please feel free to fix the idea I'm trying to communicate.

**Comment by nermiller** [comment]
@chayim mind adding this comment to this PR I submitted: https://github.com/RediSearch/RediSearch/pull/2931? Your review would also be appreciated!

File: docs/commands/ft.search.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ O(n) for single word queries. `n` is the number of the results in the result set
 
 The time complexity for more complex queries varies, but in general it's proportional to the number of words, the number of intersection points between them and the number of results in the result set.
 
+Note: if a sort order is not specified, the return order is not guaranteed.
```
Comment (oshadmi) [inline docs/commands/ft.search.md:7]:
```suggestion
Note: if a sort order is not specified, results are sorted by:
- Without Coordinator - by the document score (ascending)
- With Coordinator - by the text/numeric value (ascending)
- For similar scores or values, sort by document ids (ascending)
```

**Comment by oshadmi** [comment]
Replaced by #2931
(commit ce849525d33a5eb08ef34973d8a9bc67f463845e)

**Pull request closed without merging**